### PR TITLE
iDay standardize doc tables

### PR DESCRIPTION
**Pull request by glenhunter**
I standardized the documentation tables on `*Page.tsx` files to use the correct `NxTable`, `NxTile`, `NxCode`, `NxTextLink`, and `NxP` tags. 

I did not do the lists because we already have tickets out for them (and I ran out of time).

### CODE REVIEW

**Comment by rpokorny** [comment]
I don't know if you'd consider this a step too far for this PR, but one thing I've been considering lately is creating some helper components to standardize, and make easier, our typical props tables and classnames tables.  Kind of like how in the CSSVariablesPage I created helper components like `ColorDocRow`, rather than repeating that row structure for each row.  I'm thinking we could have something similar but shared across all of our prop tables - `GalleryPropsTable` which encapsulates the table, table head (including the actual header cells), and table body, and then you give it `GalleryPropsRow` components as children, which each encapsulate a table row.

**Comment by rpokorny** [comment]
The Additional Resources page has a bunch of bare `<a>` tags that should be `<NxTextLink>`
image

**Comment by glenhunter** [comment]
> The Additional Resources page has a bunch of bare `<a>` tags that should be `<NxTextLink>`
> 
> Edit: and a few on the font-size page and the Styling Components page and the Validation Utils page

https://github.com/sonatype/sonatype-react-shared-components/pull/459/commits/0bb9ebe0a56b38d1a36bb6bce12f6dc31bd9be5f

**Comment by glenhunter** [comment]
> I don't know if you'd consider this a step too far for this PR, but one thing I've been considering lately is creating some helper components to standardize, and make easier, our typical props tables and classnames tables. Kind of like how in the CSSVariablesPage I created helper components like `ColorDocRow`, rather than repeating that row structure for each row. I'm thinking we could have something similar but shared across all of our prop tables - `GalleryPropsTable` which encapsulates the table, table head (including the actual header cells), and table body, and then you give it `GalleryPropsRow` components as children, which each encapsulate a table row.

A step too far for this I-day PR

**Comment by rpokorny** [comment]
You missed the "Examples" files such as `NxTableExamples.tsx`.  Note that I don't mean the actual example files, but the ones where we separate out the list of examples from the rest of the Page file when there's a lot of them.

**Comment by rpokorny** [comment]
Missed `.nx-tile` and related elements in `NxViewportSizedPage`

**Comment by glenhunter** [comment]
> Missed `.nx-tile` and related elements in `NxViewportSizedPage`

https://github.com/sonatype/sonatype-react-shared-components/pull/459/commits/5ed6df8e052f5567b5266e5dcd0125a5a9e08c58

**Comment by glenhunter** [comment]
> You missed the "Examples" files such as `NxTableExamples.tsx`. Note that I don't mean the actual example files, but the ones where we separate out the list of examples from the rest of the Page file when there's a lot of them.

https://github.com/sonatype/sonatype-react-shared-components/pull/459/commits/5ed6df8e052f5567b5266e5dcd0125a5a9e08c58

File: gallery/src/styles/NxGrid/NxGridExamples.tsx [hunk 1]
```diff
@@ -5,10 +5,11 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 import React from 'react';
+
 import { GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+import { NxCode } from '@sonatype/react-shared-components';
```
Comment (rpokorny) [inline gallery/src/styles/NxGrid/NxGridExamples.tsx:10]: I see a bunch of places in this PR where you reordered imports.  If you're going to do that, what we really try to do is have imports from external modules at the top.  And that includes, from the gallery's perspective, the RSC library.

Comment (glenhunter) [reply gallery/src/styles/NxGrid/NxGridExamples.tsx:10]: > I see a bunch of places in this PR where you reordered imports. If you're going to do that, what we really try to do is have imports from external modules at the top. And that includes, from the gallery's perspective, the RSC library.

Initial effort: https://github.com/sonatype/sonatype-react-shared-components/pull/459/commits/a63a568ea2f6e9ca27a8e67773ee8ccfd864826a

Revised effort: https://github.com/sonatype/sonatype-react-shared-components/pull/459/commits/88d3650c12b1cac9aaef616ded454f759b71fabd

File: gallery/src/components/NxFormGroup/NxFormGroupPage.tsx [hunk 2]
```diff
@@ -6,7 +6,7 @@
  */
 import React from 'react';
 import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
-import { NxTableHead, NxTableRow, NxTableCell, NxTable, NxTableBody, NxTextLink }
+import { NxTable, NxTextLink, NxP, NxCode }
   from '@sonatype/react-shared-components';
```
Comment (rpokorny) [inline gallery/src/components/NxFormGroup/NxFormGroupPage.tsx:10]:
```suggestion
import { NxTable, NxTextLink, NxP, NxCode } from '@sonatype/react-shared-components';
```

**Pull request merged**